api4/team_test.go

```diff
@@ -2149,14 +2147,13 @@ func TestUpdateTeamScheme(t *testing.T) {
 	th.App.SetPhase2PermissionsMigrationStatus(true)
 
 	team := &model.Team{
-		DisplayName:     "Name",
-		Description:     "Some description",
-		CompanyName:     "Some company name",
-		AllowOpenInvite: false,
-		InviteId:        "inviteid0",
-		Name:            "z-z-" + model.NewId() + "a",
-		Email:           "success+" + model.NewId() + "@simulator.amazonses.com",
-		Type:            model.TEAM_OPEN,
+		DisplayName: "Name",
+		Description: "Some description",
+		CompanyName: "Some company name",
+		IsPublic:    false,
+		InviteId:    "inviteid0",
+		Name:        "z-z-" + model.NewId() + "a",
+		Email:       "success+" + model.NewId() + "@simulator.amazonses.com",
```

Comment:
Is there a test that covers the backwards-compatibility case somewhere?

Reply:
Not yet, this PR is not ready to review, actually.